.github/workflows/pr-review-requester.yml

```diff
@@ -0,0 +1,19 @@
+name: pr-review-requester
+
+on:
+  issues:
+    types: [opened]
```

Comment:
The auto request review doesn't do anything for issues. 
There are some other actions that can help with that if you want. 
But for this you can just remove this.

Reply:
dacf27e36da6dfadd754588e204a4fc5152838e7 done! thank you